Add Lambda proxy for asynchronous call to synchronous Lambda invoke

andrewshkovskii opened the pull request.

andrewshkovskii committed:
Add Lambda proxy.

andrewshkovskii committed:
Fix docstring

andrewshkovskii committed:
Reformat imports

andrewshkovskii committed:
Rename file since it has conflict with keyword

andrewshkovskii committed:
Little code refactor

homm commented on boto_addins/lambda_.py:
```diff
@@ -0,0 +1,128 @@
+
+    def _sign_request(self, request):
+        """Sign request to AWS with SigV4Auth."""
+        url = urlparse(request.url)
```
Пользуясь случаем советую использовать мою https://pypi.python.org/pypi/YURL

Основное что нужно:
```
Парсинг: url = URL('http://google.com:80')
Замена: url.replace(query='')
Приведение к строке: str(url)
```

homm reviewed (approved):
Остальное ок

andrewshkovskii committed:
Remove url parsing

Add YURL for url building;
Made _sing_request private since it is only for inner usage;

andrewshkovskii commented:
Избавился от разбора URL в пользу создания его через YURL.
Добавить квотирование значения параметра `qualifier` т.к. этого требует амазоновский sigv4.
($LATEST -> %24LATEST)

andrewshkovskii commented:
@homm Только смерджить надо тебе, т.к. это PR из моего форка.

homm commented:
@andrewshkovskii Я дал тебе права. Можешь межрить?
